app/controllers/publishers/vacancies/job_applications_controller.rb

```diff
@@ -32,36 +33,36 @@ def download_pdf
   end
 
   def tag_single
-    prepare_to_tag([params.fetch(:id)], "all")
+    form_key = ActiveModel::Naming.param_key(Publishers::JobApplication::TagForm)
+    params[form_key] = {
+      origin: JobApplicationsHelper::JOB_APPLICATION_DISPLAYED_STATUSES.first,
+      job_applications: [params.fetch(:id)],
+    }
```

Comment:
again I find this awkward altering the params hash rather than passing parameters explicitly - I find it very hard to follow. Not sure I understand why we are passing JobApplicationsHelper::JOB_APPLICATION_DISPLAYED_STATUSES.first as the origin when we should always be getting it from the request...?